config/account.go

```diff
@@ -82,6 +95,34 @@ func (pf *AccountPriceFloors) validate(errs []error) []error {
 		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.max_schema_dims should be between 0 and 20`))
 	}
 
+	if pf.Fetcher.Period > pf.Fetcher.MaxAge {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.period_sec should be less than account_defaults.price_floors.fetch.max_age_sec`))
+	}
+
+	if pf.Fetcher.Period < 300 {
```

Comment:
The code as written, I don't think will send more than one request every 5 minutes for a given heap element if this value is less than 300:
```
209         case <-ticker.C:
210             currentTime := f.time.Now().Unix()
211             for top := f.fetchQueue.Top(); top != nil && top.FetchTime <= currentTime; top = f.fetchQueue.Top() {
212                 nextFetch := heap.Pop(&f.fetchQueue)
213                 f.submit(nextFetch.(*FetchInfo))
214             }
floors/fetcher.go
```
If less than 5 minutes, the element will evaluate to `top.FetchTime <= currentTime` every time the `<-ticker` sends a signal. Do we need this check?

Reply:
I did not get it, can you please explain bit more on this

Reply:
It was better explained in my first comment above. No need to remove the check.